oauth2/oauth2-resource-server/src/main/java/org/springframework/security/config/annotation/web/configurers/oauth2/resourceserver/OAuth2ResourceServerConfigurer.java

```diff
@@ -0,0 +1,140 @@
+package org.springframework.security.config.annotation.web.configurers.oauth2.resourceserver;
+
+import org.springframework.security.config.annotation.web.HttpSecurityBuilder;
+import org.springframework.security.config.annotation.web.configurers.AbstractHttpConfigurer;
+import org.springframework.security.oauth2.jose.jws.JwsAlgorithms;
```

Comment:
The `config` module should not depend on `jose` or `jwt` packages/modules.
Only the `oauth2-client` and `oauth2-resource-server` modules may include the `oauth2-jose` dep

Reply:
I have a concern about `JwtDecoder` that seems to be related to this comment as well. (See below)